test/simapp/state.go

```diff
@@ -78,18 +79,21 @@ func AppStateFn(cdc codec.JSONCodec, simManager *module.SimulationManager) simty
 			panic(err)
 		}
 
-		stakingStateBz, ok := rawState[stakingtypes.ModuleName]
```

Comment:
The initial application now use multi-staking genesis instead of staking genesis.